tmt-app/src/Components/MajorPage.jsx

```diff
@@ -103,6 +120,52 @@ function MajorPage() {
             </Grid>
           );
         })}
+        <Grid item xs={12} md={6}>
+          <Card
+            sx={{
+              boxShadow: "none",
+              backgroundColor: "Avacado.main",
+              height: "100%",
+              boxSizing: "border-box",
+            }}
+          >
+            <Typography variant="h3" sx={{ paddingBottom: "1.5rem" }}>
+              Any extra resources?
+            </Typography>
+            <Typography variant="body">
+              <List>
+                {resourcesLink.map((resource, index) => {
```

Comment:
This would make more sense being plural as just resources if the iterable is `resource`